README.md

```diff
@@ -516,6 +448,76 @@ user's context within your application. Future versions of this product will pro
 
 **Note:** At this time, when device orientation changes any currently visible form is closed and will not be re-displayed automatically.
 
+## Deep Linking
+[Deep Links](https://help.klaviyo.com/hc/en-us/articles/14750403974043) allow you to navigate to a particular
+page within your app in response to a user interaction. Klaviyo supports deep linking from tapping on a Push Notification
```

Comment:
I moved this to an h2 section with almost no changes, just a few updates to mention both forms and notifications

Reply:
> Klaviyo supports deep linking from tapping on a Push Notification

should we update this to include IAFs

Reply:
yea, its on the next line

Reply:
> and from In-App Forms interactions. There are broadly three steps to implement deep links in your app: